cmd/kube-apiserver/app/options/options.go

```diff
@@ -162,6 +164,9 @@ func (s *ServerRunOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.SSHKeyfile, "ssh-keyfile", s.SSHKeyfile,
 		"If non-empty, use secure SSH proxy to the nodes, using this user keyfile")
 
+	fs.StringSliceVar(&s.SSHPreferredAddressTypes, "ssh-preferred-address-types", s.SSHPreferredAddressTypes,
+		"List of the preferred NodeAddressTypes to use for SSH proxy. Possible values: "+strings.Join(api.AllNodeAddressTypes.Names(), ", "))
```

Comment:
nit: to sue for the SSH proxy to the nodes

Reply:
done